pytorch_lightning/trainer/training_loop.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, trainer, multiple_trainloader_mode):
         self.automatic_optimization = True
         self._curr_step_result = None
         self._cur_grad_norm_dict = None
+        self.do_backward = True
```

Comment:
Minor nit: do we want this to be `_do_backward` to be consistent with the other contexts?